app/components/search/secondary_subject_selection_component.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= content_for :page_title, I18n.t('subjects.secondary_title') %>
```

Comment:
We want validation errors to appear in the `<title>`element

```
<%= content_for :page_title, title_with_error_prefix(I18n.t('subjects.secondary_title'), form.errors.any?) %>
```

Reply:
returns ` undefined method "title_with_error_prefix"`

Reply:
Have pushed a fix @ThatRandomAdmin 
https://github.com/DFE-Digital/find-teacher-training/pull/1110/commits/4ca132acefa6dc190129348dce7dbbc6a96c6d39